PatrolRewardService/PatrolRewardService.Tests/MutationTest.cs

```diff
@@ -133,7 +133,7 @@ public async Task RetryTransaction()
         await context.RewardPolicies.AddAsync(policy);
         await context.Avatars.AddAsync(avatar);
         // await context.SaveChangesAsync();
-        var configOptions = new GraphqlClientOptions {Host = "http://9c-internal-validator-5.nine-chronicles.com", Port = 80};
+        var configOptions = new GraphqlClientOptions {Host = "http://9c-internal-validator-5.nine-chronicles.com", Port = 80, JwtIssuer = "issuer", JwtSecret = "onsolhjcqbrawkvznmhuukoqunyzyigmwfixgqwvnlqlbpvqfvhfcyslwmqerpyihowcyiksouulydbuuuvlgpfskhzrcrsjorqkwnfxkkosvkkdwcxhjitwyxbfezig"};
```

Comment:
Since the necessary execution parameters have been added, there does not seem to be a need to test on actual nodes. i will add a test case for checking the header of GraphQLHttpRequestWithAuth.